wire-ios/Wire-iOS/Sources/UserInterface/Helpers/UIAlertAction+Convenience.swift

```diff
@@ -39,13 +39,15 @@ extension UIAlertAction {
     static func link(
         title: String,
         url: URL,
-        presenter: UIViewController?
+        presenter: UIViewController?,
+        onDismiss: (() -> Void)? = nil
     ) -> Self {
         return .init(
             title: title,
             style: .default
         ) { [weak presenter] _ in
             let browserViewController = BrowserViewController(url: url)
+            browserViewController.onDismiss = onDismiss
```

Comment:
Is this really required?
The `BrowserViewController.onDismiss` closure is called in `UIViewController.dismiss(animated:completion:)` but I don't think any `BrowserViewController/SFSafariViewController` should present other view controllers modally.

Reply:
I need this to soft block the app upon certificate exipiration